### DISCUSSION TITLE
语音转文本，当音频时长来到五十分钟就会失败，后续会支持直接上传音频字幕吗？

### DISCUSSION THREAD

**Discussion by girlfriendandme** [ZuodaoTech/everyone-can-use-english#628] [Q&A]
时长短的音频可以成功， 时长来到五十多分钟就会失败，有声书五十多分钟一个音频，而且还是转录二十多分钟后弹出失败。如下图。 （失败的时候会在上面弹出一段红字，来不及截图，大概意思是whisper缺少什么属性。但是转录时长短的音频不会有问题）——希望后续可以支持上传音频字幕，因为剪映这款软件也有识别字幕，速度会快很多，准确度也不比whisper 中等模型差。所以能直接上传音频字幕就省事多了，不用等那么久，结果显示失败。
![image](https://github.com/xiaolai/everyone-can-use-english/assets/48598963/de0e2310-452e-4599-9e41-68fec44b96a3)

**Comment by girlfriendandme** [comment]
后续反馈：small能完成50分钟长的音频转录，small-en以及medium跟medium-en都中途失败。

**Comment by an-lee** [comment]
嗯，后续会支持一下直接上传音频字幕
